### PR TITLE
Created a new method to get the minimum pegin tx value abstracting th…

### PR DESCRIPTION
**Pull request by nathanieliov**
…e caller of choosing what min pegin to use, whether legacy or current.

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java [hunk 1]
```diff
@@ -93,4 +93,32 @@ void test_getFederationActivationAge(BridgeConstants bridgeConstants, boolean is
...
+    }
+
+    @ParameterizedTest()
+    @MethodSource("minimumPeginTxValueArgProvider")
+    void test_getMinimumPeginTxValue(BridgeConstants bridgeConstants, boolean isRSKIP219Active){
+        // Arrange
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP383)).thenReturn(isRSKIP219Active);
+        // Act
+        long federationActivationAge = bridgeConstants.getFederationActivationAge(activations);
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java:115]: `federationActivationAge`? Careful with the copy paste

**Review by marcos-iov — CHANGES_REQUESTED** [review]

File: rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java [hunk 2]
```diff
@@ -93,4 +93,32 @@ void test_getFederationActivationAge(BridgeConstants bridgeConstants, boolean is
...
+
+        // assert
+        if (isRSKIP219Active){
+            assertEquals(bridgeConstants.federationActivationAge, federationActivationAge);
+        } else {
+            assertEquals(bridgeConstants.federationActivationAgeLegacy, federationActivationAge);
+        }
```
Comment (kelvinator07) [inline rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java:122]: You Mean `legacyMinimumPeginTxValue` And `minimumPeginTxValue` ?

File: rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java [hunk 3]
```diff
@@ -93,4 +93,32 @@ void test_getFederationActivationAge(BridgeConstants bridgeConstants, boolean is
...
+            Arguments.of(BridgeRegTestConstants.getInstance(), true)
+        );
+    }
+
+    @ParameterizedTest()
+    @MethodSource("minimumPeginTxValueArgProvider")
+    void test_getMinimumPeginTxValue(BridgeConstants bridgeConstants, boolean isRSKIP219Active){
+        // Arrange
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP383)).thenReturn(isRSKIP219Active);
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java]: Wow! I made a mess using the IntelliJ change list feature.

I pushed the fix.

File: rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java (hunk 1 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java]: Updated!

File: rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java (hunk 2 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java]: Wow! I made a mess using the IntelliJ change list feature.

I pushed the fix.

**Review by marcos-iov — APPROVED** [review]

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**